experimental/tools/unit-generators/vhdl/generators/handshake/blocker.py

```diff
@@ -0,0 +1,83 @@
+from generators.support.signal_manager import generate_signal_manager
+from generators.handshake.join import generate_join
+
+
+def generate_blocker(name, params):
+    # Number of input ports
```

Comment:
```suggestion
  # Number of input ports
```

Reading the code again I happened to find this🙂